mtags/src/main/scala/scala/meta/internal/pc/AutoImports.scala

```diff
@@ -48,18 +54,20 @@ trait AutoImports { this: MetalsGlobal =>
           case Some(pkg)
               if pkg.symbol != rootMirror.EmptyPackage ||
                 pkg.stats.headOption.exists(_.isInstanceOf[Import]) =>
-            val lastImport = pkg.stats
+            val lastImport: (Tree, Boolean) = pkg.stats
```

Comment:
I would use a pattern here so to avoid the tuple accessors below, which are hard to read

```suggestion
            val (lastImportOrPkg, padTop): (Tree, Boolean) = pkg.stats
```

Reply:
Ahh that's much nicer. Thanks!